tools/mo/openvino/tools/mo/convert_impl.py

```diff
@@ -301,16 +305,53 @@ def check_fallback(argv: argparse.Namespace):
 
     fallback_reasons['extensions'] = legacy_extensions_used
     fallback_reasons['transformations_config'] = legacy_transformations_config_used
+    fallback_reasons['tensorflow_custom_operations_config_update'] = tensorflow_custom_operations_config_update_used
 
     reasons = [reason for reason, is_applicable in fallback_reasons.items() if is_applicable(argv)]
     return reasons
 
 
+def update_fallback_with_conversion_error(use_new_frontend: bool, is_tf: bool, ex_msg: str, fallback_reasons: list):
+    import re
+    if not is_tf:
+        # this sort of fallback is only used by TensorFlow Frontend
+        return False
+
+    if use_new_frontend:
+        # this option forces to use new TensorFlow Frontend
+        # so it is not possible for the fallback
+        return False
+
+    # for TensorFlow FE we have a set of operations that should lead to the fallback to the legacy
+    conversion_error_re = r"^(\[TensorFlow\ Frontend\]\ Internal\ error\:\ No\ translator\ found\ for\ )(\w+)(\ node\.)$"
+    conversion_error_match = re.findall(conversion_error_re, ex_msg, re.MULTILINE)
+    fallback_operations = [
+        "ClipByValue",
+        "TensorArrayScatterV3", "TensorArrayV3", "TensorArraySizeV3", "TensorArrayGatherV3",
+        "LoopCond", "Enter", "NextIteration", "Exit",  # corresponds to TF1 While operation
+        "Switch", "Merge",  # corresponds to TF1 If and TF1 While operations
+        "TensorListLength", "TensorListReserve", "TensorListFromTensor",
+        "TensorListSetItem", "TensorListStack",  # corresponds to TF2 While operations
+        "FFT", "FFT2D", "FFT3D", "IFFT", "IFFT2D", "IFFT3D",
+        "RFFT", "RFFT2D", "RFFT3D", "IRFFT", "IRFFT2D", "IRFFT3D",
+        "Complex", "ComplexAbs", "Real", "Imag",  # corresponds to operations with complex tensors
+        "FIFOQueueV2", "QueueDequeueUpToV2", "QueueDequeueManyV2",
+        "QueueDequeue", "QueueDequeueV2", "IteratorGetNext",
+        "LookupTableInsert", "LookupTableInsertV2"  # corresponds to automatic pruning
```

Comment:
nit: Comments for each groups of operations are hard to match with actual groups, it would be better to rearrange it so that comment describing the group comes first and list of operations comes after it.

Reply:
Done